src/Annotations/Serializer/AnnotationNormalizer.php

```diff
@@ -0,0 +1,157 @@
+<?php
+
+namespace eLife\Annotations\Serializer;
+
+use eLife\ApiSdk\ApiSdk;
+use eLife\HypothesisClient\Model\Annotation;
+use League\CommonMark\Block\Element;
+use League\CommonMark\DocParser;
+use League\CommonMark\ElementRendererInterface;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Serializer\Normalizer\NormalizerAwareInterface;
+use Symfony\Component\Serializer\Normalizer\NormalizerAwareTrait;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+
```

Comment:
This class is very important as it marks the boundary between the `eLife\HypothesisClient` subdomain and the `eLife\Annotations` one. It's not to be confused with `AnnotationDenormalizer` and the others `Denormalizer`s.
Therefore it could be given a more unique name, like `eLife\Annotations\Serializer\HypothesisClientAnnotationNormalizer`.

Reply:
There's probably a better name than that.